php/ad-servers/class-ad-layers-dfp.php

```diff
@@ -342,7 +342,12 @@ public function get_settings_fields() {
 						'page_type' => new Fieldmanager_Select(
 							array(
 								'label' => __( 'Page Type', 'ad-layers' ),
-								'options' => Ad_Layers::instance()->get_page_types(),
+								'options' => array_merge(
+									array(
+										'all' => __( 'All Pages', 'ad-layers' ),
```

Comment:
There's currently a ["Default"](https://github.com/alleyinteractive/ad-layers/blob/master/php/class-ad-layers.php#L382) page type, and I think this becomes a bit confusing with that in mind. Is there any way you can merge the two options?